### PR TITLE
feat: add data JSON

### PR DESCRIPTION
**Pull request by helly-15**
https://trello.com/c/yZWgMF2S/2-create-json-with-data

Добавлен файл данных

### CODE REVIEW

File: src/data/data.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      ],
+      "gallery": [
+        {
+          "1856": "https://planetabelarus.by/upload/medialibrary/a96/a961f5c95e540f5bd0c535a127d8dbc0.jpg"
+        },
+        {
+          "1860": "https://planetabelarus.by/upload/medialibrary/65e/65ec6544129e46c7cbc56cdb76cf0e8a.jpg"
+        }
+      ],
+      "video": "https://www.youtube.com/watch?v=zOW38EpkgA8",
```
Comment (slolkunchik) [inline src/data/data.json]: it's better to use array of elements, not array of objects in this cases (biography, gallery, masterpiece)

Comment (helly-15) [reply src/data/data.json]: HI! didn't understand what you meant. 
If I make only elements, everything will be like a string and we need to do smth like that:
image

Comment (helly-15) [reply src/data/data.json]: corrected)

Comment (slolkunchik) [reply src/data/data.json]: you can merge

**Pull request merged**